src/app/components/translation/translation.component.spec.ts

```diff
@@ -12,6 +12,7 @@ import {Resource} from '../../models/resource';
 import {Page} from '../../models/page';
 import {CustomPage} from '../../models/custom-page';
 import {ResourceComponent} from '../resource/resource.component';
+import anything = jasmine.anything;
```

Comment:
What does this do? I just haven't seen `=` used with an `import` statement before. Seems like `const anything = jasmine.anything;` would be better unless I'm missing something...

Reply:
That was what RubyMine did automatically... 😬

Reply:
I don't think that's valid JS. Idk what RubyMine thinks it is doing. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Reply:
I forgot you were doing TS. I guess it does has that syntax https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require. But I think it is for older CommonJS and AMD features or some default export compatibility. You just are trying to alias something already available as a new variable. I think using `const` would be better. That isn't a real import.